### ISSUE TITLE
Run legacy migrations only once

### ISSUE THREAD

**Issue by mjustin** [cloudyrock/mongock#228]
Perhaps the [Legacy migration](https://www.mongock.io/legacy-migration) documentation will cover this once it's complete, but in my app once we've migrated from Mongobee to Mongock 4 for the first time, we wouldn't want to run any more legacy migrations.  However, each time at startup, it reruns the legacy migrations, inserting an additional `MongockSync4LegacyMigrationChangeLog` entry in the `mongockChangeLog` collection.  This means that the changelog collection is constantly growing from something that would never actually do anything more.  It would be nice to have the option to make the legacy migrations be a one-off change (`runAlways = false`), rather than run every time the app starts (`runAlways = true`).

### application.yml
```
spring:
  mongock:
    change-logs-scan-package: com.example.my.migrations
    legacy-migration:
      collection-name: dbchangelog
```

### mongockChangeLog
```
{
	"_id" : ObjectId("5f20476179f25d51e4f7315c"),
	"executionId" : "2020-07-28T11:42:25.463988-604",
	"changeId" : "mongock-legacy-migration",
	"author" : "mongock",
	"timestamp" : ISODate("2020-07-28T15:42:25.675Z"),
	"state" : "EXECUTED",
	"changeLogClass" : "com.github.cloudyrock.mongock.driver.mongodb.sync.v4.changelogs.MongockSync4LegacyMigrationChangeLog",
	"changeSetMethod" : "mongockSpringLegacyMigration",
	"executionMillis" : NumberLong(197),
	"_class" : "io.changock.driver.api.entry.ChangeEntry$1"
}
{
	"_id" : ObjectId("5f206b871ae456290b4146f3"),
	"executionId" : "2020-07-28T14:16:39.346268-918",
	"changeId" : "mongock-legacy-migration",
	"author" : "mongock",
	"timestamp" : ISODate("2020-07-28T18:16:39.414Z"),
	"state" : "EXECUTED",
	"changeLogClass" : "com.github.cloudyrock.mongock.driver.mongodb.sync.v4.changelogs.MongockSync4LegacyMigrationChangeLog",
	"changeSetMethod" : "mongockSpringLegacyMigration",
	"executionMillis" : NumberLong(55),
	"_class" : "io.changock.driver.api.entry.ChangeEntry$1"
}
```

**Comment by dieppa** [comment]
fixed in 4.1.12.
Cheers
